### ISSUE TITLE
Remove Citation Property from core metadata

### ISSUE THREAD

**Issue by igarousi** [I-GUIDE/data-catalog#45]
We may want to use subjectOf wherever we want to cite to another creative work.

**Comment by igarousi** [comment]
Update: We want to keep Citation. This is different from the subjectof, hasparts, and ispartof features. A complete example should look like this:
{
  "name": "NOAA National Water Model CONUS Retrospective Dataset, Version 2.1 in NetCDF Format",
  "description": "The complete archive of NWM input forcing and model output data version 2.1 in NetCDF format.",
  "url": "https://noaa-nwm-retrospective-2-1-pds.s3.amazonaws.com/index.html",
  "creator": {
    "@type" : "Organization",
    "name": "Office of Water Prediction (OWP)",
    "url": "https://water.noaa.gov/"
  },
  "citation": {
    "@type": "CreativeWork",
    "name" : "Simulating storm surge and compound flooding events with a creek-to-ocean model: Importance of baroclinic effects",
    "identifier" : "https://doi.org/10.1016/j.ocemod.2019.101526",
    "url": "https://www.sciencedirect.com/science/article/abs/pii/S1463500319302173?via%3Dihub",
    "creator" : {
      "@type" : "Person",
      "name": "Fei Ye",
      "email": "feiye@vims.edu",
      "affiliation": {
        "@type": "Organization",
        "name": "Virginia Institute of Marine Science",
        "url": "https://www.vims.edu"
      }
    },
    "creativeWorkStatus": {
      "@type": "DefinedTerm",
      "name": "published",
      "description": "The resource has been permanently published and should be considered final and complete."
    },
    "publisher" : {
      "@type": "Organization",
      "name" : "Elsevier"
    }
  } 
}

**Comment by horsburgh** [comment]
@igarousi - can citation be expressed as a string instead of a creative work? Can we simplify here? A string could be a url or a full bibliographic citation. I don’t think this will get used if we require this much information to add a related resource.

**Comment by igarousi** [comment]
@horsburgh We certainly can simplify this. It seems that there are two distinct situations that have caused some confusion. Let's break it down:

1. If we consider the citation property as a list of strings containing only URLs, the submission form becomes quite simple for users to enter metadata for their resources. In this case, let's say a resource that is being submitted to the data catalog identifies links to the datasets it used or is retrieved from. If these datasets are not registered to the data catalog and do not have associated metadata, then retrieving the entire metadata for this resource cannot provide any information about those datasets. As a result, we won't be able to establish connections between different resources that cite a specific dataset. 

2. However, if we aim to get detailed metadata for this resource, we should consider representing the citation field as a creative work. This does not necessarily mean that users have to manually input all the information in a single, lengthy submission form. Instead, I was thinking of a procedure where the metadata retrieval process reads the URLs in the citation field and searches for them in the data catalog. If those cited resources have been previously registered, the system can automatically retrieve all the relevant properties associated with them. In this case, the final output would be a nested JSON schema.  So, our goal should be to have the backend code support citation as creative works while not displaying all the citation fields on the registration form, as this could potentially confuse users. The fields I want to suggest keeping on the submission form are name, description, and URL which are similar to what we have for the subjectof, ispartof, and haspart properties. 

Does that make sense? I'm happy to hear more thoughts on this.

**Comment by horsburgh** [comment]
Given that we currently have very little in the catalog, I think the much more likely scenario is going to be that people will use the citation field to record relationships with things outside our catalog - e.g., published papers about the described resource. I think for a first version, supporting citation as a string should be sufficient.

**Comment by igarousi** [comment]
Fair enough! I will leave a note that citations can also be expressed as creative work in future versions.
